### PR TITLE
docs: basic usage insights page

### PR DESCRIPTION
**Pull request by abutterworth**
Not perfect, but a first pass:

image

image

### CODE REVIEW

File: dependent-usage-analyzer/index.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ function getPackageInfo(dir) {
       version: dependencies && dependencies['@edx/paragon'] ? dependencies['@edx/paragon'].version : 'unknown',
       name,
       repository,
+      folderName: dir.split('/').pop(),
```
Comment (abutterworth) [inline dependent-usage-analyzer/index.js:33]: Our `name` field in package json across projects is inconsistent. Folder names are better.

File: dependent-usage-analyzer/index.js [hunk 2]
```diff
@@ -110,7 +111,14 @@ function analyzeProject(dir, options = {}) {
...
   const packageJSONFiles = glob.sync(`${dir}/**/package.json`, { ignore: [`${dir}/**/node_modules/**`] });
-  return packageJSONFiles.map(packageJSONFile => path.dirname(packageJSONFile));
+
+  // If paragon isn't included in the package.json file then skip analyzing it
+  const packageJSONFilesWithParagon = packageJSONFiles.filter(packageJSONFile => {
+    const { dependencies } = JSON.parse(fs.readFileSync(packageJSONFile, { encoding: 'utf-8' }));
+    return dependencies['@edx/paragon'] !== undefined;
+  });
+
+  return packageJSONFilesWithParagon.map(packageJSONFile => path.dirname(packageJSONFile));
```
Comment (abutterworth) [inline dependent-usage-analyzer/index.js:121]: Avoid empty objects in the analysis.

File: www/src/pages/insights.jsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  </div>
+);
+
+export default function () {
+  return (
+    <div className="w-100">
+      <SEO title="Usage Insights" />
+      <header className="mb-5">
+        <h1>Usage Insights</h1>
+        <p>Last updated: 3-1-2021</p>
```
Comment (abutterworth) [inline www/src/pages/insights.jsx:86]: Manual for now. This should be automated when running the analysis is automated.

File: www/src/pages/insights.jsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    })
+  })
+  return accumulator;
+}, {});
+
+// Paragon version in all projects
+const ProjectsUsage = () => (
+  <div className="pt-5 mb-5">
+    <h3 className="mb-4">Projects in Open edX consuming Paragon</h3>
+    <DataTable
```
Comment (raq929) [inline www/src/pages/insights.jsx:30]: 🎉

**Comment by raq929** [comment]
I love it.

**Review by raq929 — APPROVED** [review]

**Pull request merged**